packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -285,7 +285,7 @@ location @service_requnbuffered {
     # It takes 3-4 minutes for the DC/OS package registry to validate a 10GB
     # package and respond. We relax the read timeout for all unbuffered
     # services to handle this.
-    proxy_read_timeout 300s;
+    proxy_read_timeout 600s;
```

Comment:
Have we seen that this extra time is required? Increasing the timeout provides bad usability when the service is never going to respond.

Reply:
I was basing the needed value off the comment.  Assuming that comment is based on testing large uploads, wouldn't we need to increase the value to support larger files?

Is the concern here that admin router could seem to be hung for 10 minutes before returning an error?  If so, 5 minutes is already an awfully long wait time so this would only turn an already bad thing into a worse bad thing.  Not sure what the solution for that would be.